pkg/operator/nfs/spec.go

```diff
@@ -89,7 +93,7 @@ func newStatefulSetForNFSServer(cr *nfsv1alpha1.NFSServer) *appsv1.StatefulSet {
 					Containers: []corev1.Container{
 						{
 							Name:  "nfs-server",
-							Image: "rook/nfs:master",
+							Image: image,
```

Comment:
I see another location below where the image is set that will need to be updated (around line 119).

Reply:
Modified that one as well, thanks